### ISSUE TITLE
Live points notifications

### ISSUE THREAD

**Issue by krishh-konar** [amosbastian/fpl#79]
This is a feature request. I was looking for a live points notification system which would provide realtime updates about the matchday points. What do you think about the idea? If its okay I would like to work on this.

**Comment by amosbastian** [comment]
Sounds interesting! Do you mean like a stream that would output live points to a terminal or something that would actually send desktop notifications?

**Comment by gringorichards** [comment]
It does sound interesting! Another option could be a function that takes an FPL entry, and returns the live points for that manager's team.

**Comment by krishh-konar** [comment]
I was thinking about actually sending desktop notifications, like when someone scores, it notifies you with the player whose points changed and your current live score.

**Comment by krishh-konar** [comment]
> It does sound interesting! Another option could be a function that takes an FPL entry, and returns the live points for that manager's team.

Yes, I think this can also be done with a few additions.

**Comment by amosbastian** [comment]
@gringorichards I think that could be done with @johnmadden86's code that he is currently working on #78.

**Comment by amosbastian** [comment]
@krishh-konar What I meant was how do you see someone using it? E.g. that they have to run a script that calls a function streams updates and sends notifications to them, or that they use the CLI or something else?

**Comment by krishh-konar** [comment]
I think running a script in the background which calls a something like a `notify_livescore` function is a way to do it, as it would require continuous polling for data while the gameweek is going on.  Do you have any other suggestions on how that can be implemented?

**Comment by amosbastian** [comment]
I'm not exactly sure how I'd implement it, but I guess it would be something that calls the `/live` endpoint e.g. every minute and compares it to the previous call to see if anything changed (and then notifies the user of this). I think it would be best to add this to the CLI so people can just run something like `fpl live` instead of having to run a script like `python live.py`, but I'm not sure if the CLI even works anymore lol (so it could be a lot of work).

**Comment by krishh-konar** [comment]
But wouldn't a notification system either require something running on the host all the time?
Would cli allow that? i.e. notifications stop as soon as you quit the program right?

**Comment by krishh-konar** [comment]
I was thinking maybe just providing a function that would take userid and poll for points continuously and people who want to use it can simple import the module and call the function, and it works until the script is running?

**Comment by amosbastian** [comment]
> But wouldn't a notification system either require something running on the host all the time?
> Would cli allow that? i.e. notifications stop as soon as you quit the program right?

You'd just leave your terminal open, which you'd have to do with a script as well anyway.

**Comment by johnmadden86** [comment]
Sounds like it should be separate project, some sort of client side app

**Comment by krishh-konar** [comment]
> You'd just leave your terminal open, which you'd have to do with a script as well anyway.

Well, you can run it as a background job or a daemon.... but I see you point.

**Comment by krishh-konar** [comment]
> Sounds like it should be separate project, some sort of client side app

Yes, maybe thats a good idea. I think I'll just start working on it on a different project and maybe just add the generic functions like getting player livescore etc. here
